resources/metadata/helmExecute.yaml

```diff
@@ -360,6 +360,13 @@ spec:
         scope:
           - STEPS
           - PARAMETERS
+      - name: skipParseAndRenderCPETemplate
```

Comment:
Just occured to me... Wouldn't `templateValues` with a default to `true` be the better choice of configuration here?

Reply:
Ah you mean with template as a verb? That took me a while to understand, maybe it would be better as something like `renderValuesTemplate` or `parseAndRenderValuesTemplate`? Would that be accurate to what is happening?

Reply:
I can understand that and agree that a new naming would be good! Let's go with `renderValuesTemplate` and see if there are any objections

Reply:
... any other name for the flag would be fine for me, too - as long as a user can understand that is about toggling the pre-processing of the values file(s).